### ISSUE TITLE
New to this! driving motors?

### ISSUE THREAD

**Issue by ralphxyz** [arkypita/LaserGRBL#824]
When I first opened LaserGRBL it ran my motors.

Now I can not get it to drive the motors.

I loaded a .jpg file in LaserGRBL and it runs but my motors are not driven.

Thanks for the help.

**Comment by ralphxyz** [comment]
Also I get nothing from the jog buttons!

**Comment by StuartB4** [comment]
What machine do you have?

**Comment by ralphxyz** [comment]
No "machine" just Arduino with GRBLcnc shield and a couple of stepper motors.

**Comment by StuartB4** [comment]
Like this one? 
Is there any movement at all ?
Is the power supply ok?
There is a fuse on the cnc shield if it's one of these, circled in white, is it ok?
![CNC-Shield-Top-Down-680x800](https://user-images.githubusercontent.com/16980026/71606174-9b46fa00-2b66-11ea-854e-edf6df028b54.jpg)

**Comment by StuartB4** [comment]
Also, did you try resetting the Arduino via the reset switch on the shield?

**Comment by ralphxyz** [comment]
Thanks for the replies. yes that is the cnc shield.

The fuse is good!.

Yes I did a reset from the shield

**Comment by StuartB4** [comment]
Can you post a screen shot of your grbl settings.

![sdsdsd](https://user-images.githubusercontent.com/16980026/71617833-ceb07580-2bb4-11ea-9269-6461d364e62c.jpg)

![sdsdsds](https://user-images.githubusercontent.com/16980026/71617836-d2dc9300-2bb4-11ea-972d-2ca98c5bdf2f.jpg)

**Comment by ralphxyz** [comment]
Settings:
[image: LaserGRBL Settings 1.jpg]

**Comment by ralphxyz** [comment]
Well that didn't work,let me try again. There isn't any image  options. How did you post the image?
I will try a link.

[Settings:]
(https://photos.app.goo.gl/i2UANPeeGf5bSp7cA)

**Comment by ralphxyz** [comment]
Strange, I reloaded the GRBL sketch and now it works again!!

Don't know why but glad it works!

Now I'll be able to start asking questions about actually using LaserGRBL.

**Comment by StuartB4** [comment]
Hi Ralph. Sorry for late reply, got a bit busy today. Glad you got it sorted.

One thing I would mention is to change $32 to 1 in the grbl configuration settings.
That enables Laser mode and gives you better results.

Also $130 and $131. change them to the actual engraving area/size of your machine.
For example if your machine can engrave an area of say 300mm in the X and 400mm in the Y direction, change the $130 to 300 and $131 to 400. It's just in case you want to use Continuous Jogging in the LaserGRBL's Settings menu. Which means it will jog as long as you keep the button pressed, but will stop automatically when you let go or when it reaches them numbers. So you can't over do it.

**Comment by StuartB4** [comment]
To post an image just click on "Attach files by dragging & dropping, selecting or pasting them"  at the bottom of the comment box.

**Comment by StuartB4** [comment]
If your motors/axis's move to slow change $110 and 111 to a higher number something like 10000 to 12000. 
If they take too long to get up to speed change $120 and 121, thats the (acceleration speed) to a higher number, not too high though or you will loose steps.

**Comment by StuartB4** [comment]
To change any of the above settings just start LaserGRBL and click Connect.
Click the Grbl menu and select Grbl Configuration like you did to get the settings window.
Click in the Value box for the $ number you want to change and click again to enable edit.
Change the number to the one you want and press Enter afterwards, do this for all the settings you want to change and press Enter after each one.
When you've done them all click on the Write button at the bottom to store them in the Arduino. Then Close.

**Comment by ralphxyz** [comment]
Thanks Stuart, next I need to build the machine I have a lot of things in
mind plus I have a K40 laser
with it's original controller I might substitute the GRBL CNC,

Got a lot to learn.

Thanks again,

Ralph

**Comment by StuartB4** [comment]
I'm sure you'll enjoy it. Good luck with it, and there's always someone here to help out if needed.
Also on the LaserGRBL Facebook page here: https://www.facebook.com/groups/486886768471991
